### ISSUE TITLE
config files - need explanation (fhd vs mhead vs largea vs lowa, differences between config files, how to choose ,etc.)

### ISSUE THREAD

**Issue by CDahmsSpireon** [traveller59/second.pytorch#467]
Can somebody explain the naming of the config files?

I gather `all` means detects all objects (car, pedestrian, cyclist, etc.) vs `car` means only detects cars.  Also I gather `pp` is short for PointPillars.  But what are the other abbreviations?  For example what do the following mean:

```
fhd
mhead
largea
lowa
```

I'm under the impression that if I'm using the nuScenes set I should choose a config file in `second.pytorch/second/configs/nuscenes` whereas if I'm using the Kitti set I should choose a config file in `second.pytorch/second/configs` or `second.pytorch/second/configs/pointpillars` is this correct?

Also within `second.pytorch/second/configs/pointpillars` what is `pp_pretrain.config`, does `pretrain` mean it should not be used for training?  Also in, for example, `second.pytorch/second/configs/pointpillars/car`, what are the `xyres_xx.config` files and what is the advantage/disadvantage of using them vs the others?

Some general explanation of the config file organization and naming would be greatly appreciated.

**Comment by chyohoo** [comment]
hi all, I am also much appreciated it if someone could explain this.
